lib/koans/20_comprehensions.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Comprehensions do
+  use Koans
+
+  @intro "Comprehensions"
+
+  # A comprehension is made of three parts: generators, filters and collectables.
+
+  koan "Generators provide the values to be used in a comprehension" do
+    # In the expression below, `n <- [1, 2, 3, 4]` is the generator.

```

Comment:
Similar trick as above, can we fold the key idea of the comment up into the description?